readme.md

```diff
@@ -38,7 +36,7 @@ with manual, [see more](https://github.com/react-native-community/react-native-w
 
 ## How to get `appId`, `appSecret` of instagram?
 
-> [Simple setup](https://developers.facebook.com/docs/instagram-basic-display-api/getting-started), You only need to complete the first 3 steps.
+> [Simple setup](https://developers.facebook.com/docs/instagram-basic-display-api/getting-started), you only need to complete step 3.
```

Comment:
No, my mean is the first 3 steps: 1,2,3. It's not only the third step.

Reply:
@AmirDoreh could you revert readme.md?

Reply:
sure

Reply:
readme has been updated @hungdev

Reply:
@AmirDoreh and do not remove this tag: ## [Here is a video tutorial](https://www.youtube.com/watch?v=Xw-mCiyUMgE)

Reply:
@AmirDoreh i would like to keep the tutorial in readme please don't remove it

Reply:
done

Reply:
I published new version. thank you.